src/msix/pack/MappingFileParser.cpp

```diff
@@ -0,0 +1,283 @@
+#include "MappingFileParser.hpp"
+#include <iostream>
+#include <fstream>
+#include <map>
+
+namespace MSIX {
+
+    const std::string WHITESPACE = " \t";

```

Comment:
>WHITESPACE [](start = 22, length = 10)

TAB?

Reply:
As per other implementations, I see whitespace and tab as the delimiter.